llpc/lower/llpcSpirvLowerGlobal.cpp

```diff
@@ -314,6 +314,12 @@ void SpirvLowerGlobal::handleCallInst(bool checkEmitCall, bool checkInterpCall)
             mangledName.startswith(gSPIRVName::InterpolateAtVertexAMD)) {
           // Translate interpolation functions to LLPC intrinsic calls
           auto loadSrc = callInst->getArgOperand(0);
+
+          // TODO: Remove this when LLPC will switch fully to opaque pointers.
+          // This is needed to handle additional bitcast which was added while creating function call.
+          if (auto bitCast = dyn_cast<BitCastInst>(loadSrc))
+            loadSrc = bitCast->getOperand(0);
+
```

Comment:
Can you please move this lower to just above the distinction between `GetElementPtrInst` and `GlobalVariable`? Then all the `loadSrc` handling is together in one place.